eopdev/ap-southeast-2/eopdev/services/ecs-eop-manager/terragrunt.hcl

```diff
@@ -12,29 +12,14 @@ include "envcommon" {
 }
 
 locals {
-  tls_secrets_manager_arn = "arn:aws:secretsmanager:ap-southeast-2:657968434173:secret:SampleAppBackEndCA-yT4JVB"
-  db_secrets_manager_arn  = "arn:aws:secretsmanager:ap-southeast-2:657968434173:secret:RDSDBConfig-wHo5DD"
+  config_secrets_manager_arn  = "arn:aws:secretsmanager:ap-southeast-2:657968434173:secret:EOPManagerConfig-cWXx3Q"
 
-  # List of environment variables and container images for each container that are specific to this environment. The map
-  # key here should correspond to the map keys of the _container_definitions_map input defined in envcommon.
-  service_environment_variables = {
-    (include.envcommon.locals.service_name) = [
-      {
-        name  = "CONFIG_SECRETS_SECRETS_MANAGER_TLS_ID"
-        value = local.tls_secrets_manager_arn
-      },
-      {
-        name  = "CONFIG_SECRETS_SECRETS_MANAGER_DB_ID"
-        value = local.db_secrets_manager_arn
-      },
-    ]
-  }
```

Comment:
This was all just cruft that wasn't needed (copy / pasted from the Gruntworks sample app)